pkg/backporter/backporter.go

```diff
@@ -311,40 +356,46 @@ func getClonesTemplateData(bugID int, client bugzilla.Client, allTargetVersions
 	if err != nil {
 		return nil, http.StatusNotFound, fmt.Errorf("Bug#%d not found: %w", bugID, err)
 	}
-
-	prs, err := client.GetExternalBugPRsOnBug(bug.ID)
-	if err != nil {
-		return nil, http.StatusInternalServerError, fmt.Errorf("Bug#%d - error occured while retreiving list of PRs: %w", bug.ID, err)
-	}
-
-	parent, err := client.GetRootForClone(bug)
-	if err != nil {
-		return nil, http.StatusInternalServerError, fmt.Errorf("unable to fetch root bug: %w", err)
-	}
-
 	clones, err := client.GetAllClones(bug)
 	if err != nil {
 		return nil, http.StatusInternalServerError, fmt.Errorf("unable to get clones: %w", err)
 	}
+	if len(clones) < 1 {
+		return nil, http.StatusInternalServerError, fmt.Errorf("clones list empty")
+	}
+	root := clones[0]
 	// Target versions would be used to populate the CreateClone dropdown
 	targetVersions := sets.NewString(allTargetVersions.List()...)
 	// Remove target versions of the original bug
 	targetVersions.Delete(bug.TargetRelease...)
-	for _, clone := range clones {
-		clonePRs, err := client.GetExternalBugPRsOnBug(clone.ID)
-		if err != nil {
-			return nil, http.StatusInternalServerError, fmt.Errorf("Bug#%d - error occured while retreiving list of PRs : %w", clone.ID, err)
-		}
+	g := new(errgroup.Group)
+	var prs []bugzilla.ExternalBug
+
+	g.Go(func() error {
+		prs, err = client.GetExternalBugPRsOnBug(bugID)
+		return err
+	})
 
+	for _, clone := range clones {
+		clone := clone
 		// Remove target releases which already have clones
 		targetVersions.Delete(clone.TargetRelease...)
-
-		clone.PRs = clonePRs
+		g.Go(func() error {
+			clonePRs, err := client.GetExternalBugPRsOnBug(clone.ID)
+			if err != nil {
+				return fmt.Errorf("Bug#%d - error occured while retreiving list of PRs : %w", clone.ID, err)
+			}
+			clone.PRs = clonePRs
```

Comment:
This variable is created on line 380 so from what I can see, we assign and never read that back?

Reply:
We are returning the clones variable, since clone is a pointer to an element in that array, the changes made to it get reflected when the array is returned.